Libraries/LibGUI/TabWidget.cpp

```diff
@@ -244,6 +244,13 @@ void TabWidget::mousedown_event(MouseEvent& event)
                 if (on_middle_click && widget)
                     on_middle_click(*widget);
             });
+        } else if (event.button() == MouseButton::Right) {
```

Comment:
The system has a separate concept for context menu events (by overriding `context_menu_event()`.)
We should use that here too, instead of special-casing right-click in `mousedown_event()` :)

Reply:
I didn't realize there was a specific event for that, handy!